ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -291,6 +296,7 @@ import '../nuxeo-button-styles.js';
                     checked$="[[selected]]"
                     on-click="_onCheckBoxTap"
                     on-keydown="_onCheckBoxKeydown"
+                    disabled$="[[_isSelectAllActive]]"
```

Comment:
Shouldn't this be 
```suggestion
                    disabled$="[[selectAllActive]]"
```
?

Reply:
It can be both because the iron-data-table has the `nuxeo-page-provider-display-behavior` (see [here](https://github.com/nuxeo/nuxeo-elements/blob/ELEMENTS-1337-select-all-page-provider-display-behavior/ui/nuxeo-page-provider-display-behavior.js#L60)), and we expose the `selectAllActive` as a computed property (see [here](https://github.com/nuxeo/nuxeo-elements/blob/ELEMENTS-1337-select-all-page-provider-display-behavior/ui/nuxeo-page-provider-display-behavior.js#L136)).
I opted for this solution, so users couldn't set the selectAllActive value directly using the API, wdyt?

Reply:
This applies to other places inside the `nuxeo-page-provider-display-behavior` where we manipulate the `_isSelectAllActive`.